guppylang/nodes.py

```diff
@@ -387,12 +387,3 @@ def __init__(
         self.cfg = cfg
         self.ty = ty
         self.captured = captured
-
-
-class CopyNode(ast.expr):
-    value: "PlaceNode"
-
-    def __init__(self, val: "PlaceNode"):
-        self.value = val
-
-    fields = ("value",)
```

Comment:
Don't treat `copy` as its own special AST node. Just use a regular call and linearity checking will magically work on its own